overrides/common/story-data.ts

```diff
@@ -0,0 +1,35 @@
+export const themeLandingPageIds = [
+  "air_quality",
+  "agriculture",
+  "biodiversity",
+  "disasters",
+  "energy",
+  "greenhouse_gases",
+  "sea_level_rise",
+  "wildfires",
+  "water_resources",
+];
+
+export const teachToolsStoryIds = [
+  "globe_protocol_etraining",
+  "stem_resources",
+  "climate_kids",
+];
+
+export const teachK12StoryIds = [
+  "globe_protocol_etraining",
+  "stem_resources",
+  "climate_kids",
+];
+
+export const visitLocationStoryIds = [
+  "globe_protocol_etraining",
+  "stem_resources",
+  "climate_kids",
+];
+
+export const visitLocationFeaturesStoryIds = [
+  "globe_protocol_etraining",
+  "stem_resources",
+  "climate_kids",
```

Comment:
❓ : Why do these repeat so much? If this is common across, we can set it once to a more general var like `const commonStoryIds` 

but looks like only `teachK12StoryIds` looks like its being used, if we aren't using the others right now, can we get rid of it and have them resurface in a separate PR when they are being used ?

Reply:
These are placeholders. I put these ids for dev purpose. All of them will hold different ids and  @agurvich will fill this up later, I just made a space. Story ids for `visit~` will be used in 'Visit' page.